app/steps/step_navigation.rb

```diff
@@ -35,6 +35,7 @@ class StepNavigation
     ],
     "Preferences" => [
       PreferencesInterviewController,
+      PreferencesAnythingElseController,
```

Comment:
do we think this controller should be considered part of the preferences section? Feels kinda separate to me

Reply:
Agreed. Renamed `Preferences` to `General` in a1b092f